cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/MdsKey.java

```diff
@@ -17,9 +17,14 @@
 package co.cask.cdap.data2.metadata.dataset;
```

Comment:
I don't think the changes in this class would be backwards compatible - would they?